modules/nw-egress-ips-about.adoc

```diff
@@ -20,6 +20,8 @@ To ensure that you can reliably allow access to the server from only that specif
 
 An egress IP address assigned to a namespace is different from an egress router, which is used to send traffic to specific destinations.
 
+When application pod and ingress router pod run on the same node and you configured an egress IP for the application project, the egress IP is not used if you send a request to a route from the application project.
```

Comment:
You could consider breaking this sentence into two to enhance readability and help with localisation. For example:

"In some cluster configurations, application pods and ingress router pods run on the same node. If you configure an egress IP for an application project in this scenario, the IP is not used when you send a request to a route from the application project."

Reply:
Addressed

Reply:
@pneedle-rh Thanks for the feedback. I addressed the change.